cmd/evm/internal/t8ntool/execution.go

```diff
@@ -303,11 +345,41 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		amount := new(big.Int).Mul(new(big.Int).SetUint64(w.Amount), big.NewInt(params.GWei))
 		statedb.AddBalance(w.Address, amount)
 	}
+	if chainConfig.IsPrague(big.NewInt(int64(pre.Env.Number)), pre.Env.Timestamp) {
+		if err := overlay.OverlayVerkleTransition(statedb, common.Hash{}, chainConfig.OverlayStride); err != nil {
+			log.Error("error performing the transition", "err", err)
```

Comment:
Can we return an error or do something more blocking?
If the state conversion fails, we should definitely get to know somehow instead of only logging an error.